packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -875,7 +874,7 @@ export function injectIntoDevTools(devToolsConfig: DevToolsConfig): boolean {
     setErrorHandler,
     setSuspenseHandler,
     scheduleUpdate,
-    currentDispatcherRef: ReactCurrentDispatcher,
+    currentDispatcherRef: ReactSharedInternals,
```

Comment:
rename this so it's clear it's the entire Internals object?

Reply:
I'm not sure if this is versioned or not.

Reply:
This is versioned so it's not so easy to rename.

Reply:
Would need to change the name but really should we do that every time we change any field on this too? Seems like it can change format many times without changing names.